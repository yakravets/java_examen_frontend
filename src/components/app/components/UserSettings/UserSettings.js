import React from 'react';
import { PageHeader } from 'antd';
import { FcSettings } from "react-icons/fc";

export default class UserSettigs extends React.Component {

    constructor(props){
        super(props);
        this.state = {name: this.props.name}
    }

    componentDidMount(){
        
    }
    render(){        
        return (
            <>
                <PageHeader
                    className="site-page-header"
                    avatar={{ src: <FcSettings /> }}
                    title="My settings:"
                    subTitle={this.state.name}
                />
            </>
        );
    }
}