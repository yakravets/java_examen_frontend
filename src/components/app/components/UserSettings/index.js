import UserSettigs from "./UserSettings";
export default UserSettigs;