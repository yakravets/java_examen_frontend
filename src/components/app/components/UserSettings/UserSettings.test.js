import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import toJson from "enzyme-to-json";

import UserSettings from './UserSettings';

it("renders correctly", () => {
    const tree = shallow(<UserSettings />);
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow(<UserSettings />);
});