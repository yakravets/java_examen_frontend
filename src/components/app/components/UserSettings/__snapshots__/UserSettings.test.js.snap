// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <PageHeader
    avatar={
      Object {
        "src": <FcSettings />,
      }
    }
    className="site-page-header"
    title="My settings:"
  />
</Fragment>
`;
